pkg/controller/jobs/job/job_controller.go

```diff
@@ -271,6 +300,15 @@ func (j *Job) podsCount() int32 {
 	return podsCount
 }
 
+func (j *Job) minPodsCount() *int32 {
+	if strVal, found := j.GetAnnotations()[JobMinParallelismAnnotation]; found {
+		if iVal, err := strconv.Atoi(strVal); err == nil {
+			return pointer.Int32(int32(iVal))
+		}
+	}
```

Comment:
Can we output a log for parsing errors as `logger.V(3).Info()`? It would be helpful for debugging.